runtime/src/main/resources/application.yml

```diff
@@ -80,3 +80,11 @@ generator:
 controllers:
   integration:
     enabled: true
+
+client:
+  state:
+    authenticationAlgorithm: HmacSHA1
```

Comment:
Did you mean to add these as default settings? How about leaving these out of here?